clang/docs/OpenMPSupport.rst

```diff
@@ -490,7 +490,7 @@ implementation.
 +-------------------------------------------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------+
 | memscope clause for atomic and flush                        | :none:`unclaimed`         | :none:`unclaimed`         |                                                                          |
 +-------------------------------------------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------+
-| transparent clause (hull tasks)                             | :none:`unclaimed`         | :none:`unclaimed`         |                                                                          |
+| transparent clause (hull tasks)                             | :part:`partial`           | :none:`unclaimed`         |  https://github.com/llvm/llvm-project/pull/166810                        |
```

Comment:
Please add "Clang Parsing/Sema:" or something similar to make it clear which part is done.